sdk/vision/ai-vision-image-analysis-rest/review/ai-vision-image-analysis.api.md

```diff
@@ -128,7 +129,7 @@ export interface CaptionResultOutput {
 }
 
 // @public
-function createClient(endpoint: string, credentials: KeyCredential, options?: ClientOptions): ImageAnalysisClient;
+function createClient(endpointParam: string, credentials: TokenCredential | KeyCredential, { apiVersion, ...options }?: ImageAnalysisClientOptions): ImageAnalysisClient;
```

Comment:
Why has this changed to endpointParam? If needed, it's another breaking change that needs to be document in CHANGELOG.md

Reply:
JS doesn't support named parameters in callers... How does this break?